tests/test_validate_data.py

```diff
@@ -0,0 +1,75 @@
+# tests/test_validate_diabetes_data.py
+# author: Jessica Kuo
+# date: 2024-12-11
+
+import pytest
+import os
+import numpy as np
+import pandas as pd
+import pandera as pa
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.validate_diabetes_data import validate_diabetes_data
+
+# Valid data example
+valid_data = pd.DataFrame({
```

Comment:
For all valid data columns where `nullable = True`, at least one null value should be included so that when we run tests we know nulls are passable.